test/testcafe/messagelist_tests.ts

```diff
@@ -146,10 +149,31 @@ const clickChatByName = async (t, name) => {
   )
 }
 
-async function clickAppMenuItem(t, label) {
-  await t.click('#main-menu-button')
-  await t.expect(Selector('a.bp3-menu-item').withText(label).exists).ok()
-  await t.click(Selector('a.bp3-menu-item').withText(label))
+export async function clickThreeDotMenuItem(t, label) {
+  await t.click('#three-dot-menu-button')
+  await t
+    .expect(Selector('.dc-context-menu > .item').withText(label).exists)
+    .ok()
+  await t.click(Selector('.dc-context-menu > .item').withText(label))
+}
+
+export async function clickSideBarItem(t, label) {
+  await t.click('#hamburger-menu-button')
+  await t.expect(Selector('.sidebar-item').withText(label).exists).ok()
+  await t.click(Selector('.sidebar-item').withText(label))
+}
+
+export async function goToSideBarSubSettingsMenu(t, label) {
+  await clickSideBarItem(t, await translate('menu_settings'))
+  await t.expect(Selector('.SettingsIconButton').withText(label).exists).ok()
+  await t.click(Selector('.SettingsIconButton').withText(label))
+}
+
+export async function sendVideoChatInvitation(t) {
+  await t.click('#test-attachment-menu')
+  await t.click(
+    Selector('a.bp3-menu-item').withText(await translate('videochat'))
+  )
```

Comment:
You can put functions that you are reusing to helpers.js :)

Reply:
maybe would make some sense to convert all tests to TS in that case?

Reply:
> there are some message list tests regarding video chat that should fail, because there is a bug in desktop and the test should detect that if it is working correctly

Lets keep the needed changes for this pr simple so we can merge soon?